Android/app/src/main/res/values/styles.xml

```diff
@@ -37,6 +37,10 @@
 
     <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light" />
 
+    <style name="ToolbarText" parent="@style/TextAppearance.Widget.AppCompat.Toolbar.Title">
+        <item name="android:textSize">17sp</item>
```

Comment:
change this textSize to 18sp instead of 17sp.
reference - https://stackoverflow.com/questions/9133290/what-is-the-default-actionbar-title-font-size

Reply:
done